snapshots/input/syntax/src/destructuring.ts

```diff
@@ -0,0 +1,33 @@
+interface Props {
+  a: number
+}
+const props: Props = { a: 42 }
+
+export function objectDestructuring(): number[] {
```

Comment:
suggestion: Add a test for destructuring in function parameter lists

Reply:
Good idea. Done, and it works out of the box